### ISSUE TITLE
[Bug] If bot stops after a street replay, it won't start properly

### ISSUE THREAD

**Issue by curtiola** [will7200/Yugioh-bot#37]
The bot stopped at the end of a street replay, on the victory screen with the OK button, & 4 hours later when it tried to run again it wasn't looking for OK & just got stuck there.

2018-01-21 12:22 - INFO - apscheduler.executors.default - Running job "DuelLinkRunTime.main.<locals>.in_main (trigger: date[2018-01-21 12:22:30 MST], next run at: 2018-01-21 12:22:30 MST)" (scheduled at 2018-01-21 12:22:30.944336-07:00)
2018-01-21 12:22 - INFO - apscheduler.scheduler - Removed job cron_main_at_2018-01-21T12:22:30.944336
2018-01-21 12:22 - DEBUG - bot - Value last_run_at modified to 2018-01-21 12:22:30.961627
2018-01-21 12:22 - INFO - apscheduler.scheduler - Added job "DuelLinkRunTime.dump" to job store "default"
2018-01-21 12:22 - DEBUG - bot - Timeout dump Scheduled
2018-01-21 12:22 - DEBUG - bot.provider.Nox - LOOKING FOR BACK BUTTON, HIGH CORRERLATION
2018-01-21 12:22 - INFO - bot - starting auto play through
2018-01-21 12:22 - INFO - bot.provider.Nox - starting auto run through
2018-01-21 12:22 - DEBUG - bot.provider.Nox - Run through 1
2018-01-21 12:22 - DEBUG - bot.provider.Nox - LOOKING FOR BACK BUTTON, HIGH CORRERLATION
2018-01-21 12:22 - INFO - apscheduler.executors.default - Running job "DuelLinkRunTime.dump (trigger: date[2018-01-21 12:22:35 MST], next run at: 2018-01-21 12:22:35 MST)" (scheduled at 2018-01-21 12:22:35.965141-07:00)
2018-01-21 12:22 - INFO - apscheduler.scheduler - Removed job e757313790954141818bd54cf8339523
2018-01-21 12:22 - DEBUG - bot - Dump Getting Called {'battle_calls': {'beforeStart': [], 'afterStart': [], 'beforeEnd': [], 'afterEnd': []}, 'last_run_at': datetime.datetime(2018, 1, 21, 12, 22, 30, 961627), 'next_run_at': datetime.datetime(2018, 1, 21, 12, 22, 30, 944336), 'playmode': 'autoplay', 'run_now': False, 'stop': False}
2018-01-21 12:22 - INFO - apscheduler.executors.default - Job "DuelLinkRunTime.dump (trigger: date[2018-01-21 12:22:35 MST], next run at: 2018-01-21 12:22:35 MST)" executed successfully
2018-01-21 12:22 - DEBUG - bot.provider.Nox - Run through 2
2018-01-21 12:22 - DEBUG - bot.provider.Nox - LOOKING FOR BACK BUTTON, HIGH CORRERLATION
2018-01-21 12:22 - DEBUG - bot.provider.Nox - Run through 3
2018-01-21 12:22 - DEBUG - bot.provider.Nox - LOOKING FOR BACK BUTTON, HIGH CORRERLATION
2018-01-21 12:22 - DEBUG - bot.provider.Nox - Run through 4
2018-01-21 12:22 - DEBUG - bot.provider.Nox - LOOKING FOR BACK BUTTON, HIGH CORRERLATION
2018-01-21 12:22 - DEBUG - bot.provider.Nox - Run through 5
2018-01-21 12:22 - DEBUG - bot.provider.Nox - LOOKING FOR BACK BUTTON, HIGH CORRERLATION
2018-01-21 12:22 - DEBUG - bot.provider.Nox - Run through 6
2018-01-21 12:22 - DEBUG - bot.provider.Nox - LOOKING FOR BACK BUTTON, HIGH CORRERLATION
2018-01-21 12:23 - DEBUG - bot.provider.Nox - Run through 7
2018-01-21 12:23 - DEBUG - bot.provider.Nox - LOOKING FOR BACK BUTTON, HIGH CORRERLATION
2018-01-21 12:23 - DEBUG - bot.provider.Nox - Run through 8
2018-01-21 12:23 - DEBUG - bot.provider.Nox - LOOKING FOR BACK BUTTON, HIGH CORRERLATION
2018-01-21 12:23 - INFO - bot - completed auto play through
2018-01-21 12:23 - DEBUG - bot - Value next_run_at modified to 2018-01-21 16:23:11.462855
2018-01-21 12:23 - INFO - apscheduler.scheduler - Added job "DuelLinkRunTime.dump" to job store "default"
2018-01-21 12:23 - DEBUG - bot - Timeout dump Scheduled
2018-01-21 12:23 - INFO - apscheduler.scheduler - Added job "DuelLinkRunTime.main.<locals>.in_main" to job store "default"
2018-01-21 12:23 - INFO - apscheduler.executors.default - Job "DuelLinkRunTime.main.<locals>.in_main (trigger: date[2018-01-21 12:22:30 MST], next run at: 2018-01-21 12:22:30 MST)" executed successfully
2018-01-21 12:23 - INFO - apscheduler.executors.default - Running job "DuelLinkRunTime.dump (trigger: date[2018-01-21 12:23:16 MST], next run at: 2018-01-21 12:23:16 MST)" (scheduled at 2018-01-21 12:23:16.465358-07:00)
2018-01-21 12:23 - INFO - apscheduler.scheduler - Removed job d96c8a05ba094a2d9d24b59308b3e1d9
2018-01-21 12:23 - DEBUG - bot - Dump Getting Called {'battle_calls': {'beforeStart': [], 'afterStart': [], 'beforeEnd': [], 'afterEnd': []}, 'last_run_at': datetime.datetime(2018, 1, 21, 12, 22, 30, 961627), 'next_run_at': datetime.datetime(2018, 1, 21, 16, 23, 11, 462855), 'playmode': 'autoplay', 'run_now': False, 'stop': False}
2018-01-21 12:23 - INFO - apscheduler.executors.default - Job "DuelLinkRunTime.dump (trigger: date[2018-01-21 12:23:16 MST], next run at: 2018-01-21 12:23:16 MST)" executed successfully

**Comment by will7200** [comment]
Weird, since I implemented a checker for street replay only for the second page. Does your street replay exist on the second page? I will investigate further.

**Comment by curtiola** [comment]
Yeah it's on the PvP page in game, wild guess: maybe somehow because I hadn't shut down the bot program for at least 30 hours, it may have been unsure of the page it's on. 
Not sure how it detects the 2nd page, is there a specific screen I should be starting the bot on?

**Comment by will7200** [comment]
Yeah, the bot tracks the location of the current page by the arrows at the bottom of the game. This gets updated for every possible battle location. Therefore there is no need to start on a particular page. This has been included in since the early days.

**Comment by curtiola** [comment]
It's only happened to me those 2 times, so it must be working fine for the most part, maybe just didn't do the page check before looking for the OK button for some reason.

**Comment by will7200** [comment]
Hopefully fixed in d4f704d0615b2372fb30bfa0ef57f23592b68ade
